Sprint-1/implement/max.test.js

```diff
@@ -16,28 +16,51 @@ const findMax = require("./max.js");
 // When passed to the max function
 // Then it should return -Infinity
 // Delete this test.todo and replace it with a test.
-test.todo("given an empty array, returns -Infinity");
+test("an empty array should return -Infinity", () => {
+  expect(findMax([])).toBe("-Infinity");
```

Comment:
No.  `-Infinity` is a real value in JavaScript and it is **not** a `String`.  You should have this as the test:
`expect(findMax([])).toBe(-Infinity);`